ai/executors/debug_executor.py

```diff
@@ -33,39 +32,23 @@ def exec(self) -> None:
             self._send_books()
             self.last_time = time.time()
 
-    def _send_books(self) -> None:
-        cmd_tactics = {'strategy': PlayState().strategy_book.strategies_name,
-                       'strategy_default': PlayState().strategy_book.default_strategies,
-                       'tactic': PlayState().tactic_book.tactics_name,
-                       'tactic_default': PlayState().tactic_book.default_tactics,
-                       'action': ['None']}
+    def _send_books(self):
 
-        msg = UIDebugCommandFactory().books(cmd_tactics)
+        msg = DebugCommandFactory().books(strategy_book=PlayState().strategy_book.strategies_name,
```

Comment:
I think PlayState() is not a singleton anymore.

Reply:
yup, will fix